eqllib/analytics/execution/T1047-suspicious-image-loads-via-commonly-abused-executable.toml

```diff
@@ -0,0 +1,21 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "low"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies suspicious image load events tied to Koadic stager processes."
+id = "45835773-d859-4d44-8ef6-5d95f7538612"
+name = "Suspicious Image Loads via Commonly Abused Executables"
```

Comment:
The name looks generic but the logic looks specific.
Also,  I believe that it's totally normal for mshta.exe to load most of these libraries, except `wmiutils.dll`.

I think I'll pull this one out for a second pass to get the PR in before the post publishes.